src/components/radio-button/radio-button.stories.mdx

```diff
@@ -151,7 +151,7 @@ Passing a boolean to these props will display only a properly colored border.
 
 Validation status could also be passed to the `RadioButtonGroup` component as a string or boolean. 
 
-For more information check our [Validations](?path=/docs/documentation-validations--radio-button-validation#grouped-inputs-examples) documentation page
+For more information check our [Validations](?path=/docs/documentation-validations--grouped-input-validation#grouped-inputs) documentation page
```

Comment:
```suggestion
For more information check our <LinkTo kind="Documentation/Validations">Validations documentation page</LinkTo>.
```
What do think about this? This way the link won't likely break if the URL happens to change.

Reply:
LinkTo doesn't work for page anchors, so we could not use it in this case, as we have to point to `Grouped Inputs` paragraph.